spec/controllers/queued_videos_controller_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'spec_helper'
+
+describe QueuedVideosController do
+  describe 'GET index' do
+    context 'with authentication' do
+      let(:user) { Fabricate(:user) }
+      let!(:queue1) { Fabricate(:queued_video, user: user) }
+      let!(:queue2) { Fabricate(:queued_video, user: user) }
+
+      before do
+        session[:user_id] = user.id
+        get :index
+      end
+
+      it 'sets @queued_videos if user authenticated' do
+        expect(assigns(:queued_videos)).to eq([queue1, queue2])
+      end
+
+      it 'renders the index template if authenticated' do
+        expect(response).to render_template(:index)
+      end
+    end
+
+    it 'redirects to /login if not authenticated' do
```

Comment:
I wouldn't hard code the path. I would say "redirects to login page" or "redirects to login path"